src/regions/ca/info/faq/43_wash-hands-home.md

```diff
@@ -0,0 +1,12 @@
+## If I stay home, should I wash my hands on a regular basis?
+
+If you're at home, you need to wash your hands:
+
+- After coughing, sneezing, blowing your nose or touching the inside of your mouth.
+- After going to the bathroom
+- Before preparing food
+- With soap and water preferably for more than 20 seconds or with an alcohol-based hand sanitizer if you do not have access to soap. Note that alcohol-based hand sanitizer is not effective if hands are soiled.
+
+This allows the prevention of any type of contamination.
+
+More information [here](https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks.html) and [here](https://www.cdc.gov/coronavirus/2019-ncov/prepare/prevention.html).
```

Comment:
This is a link to the GH repo, maybe a link to the FAQ page instead?

Reply:
Fixed